imesec-full-stack.yml

```diff
@@ -83,6 +83,21 @@ services:
       placement:
         constraints: [node.role == manager]
 
+  bind:
+    container_name: bind
+    image: sameersbn/bind:9.11.3-20190706
+    environment:
+      WEBMIN_ENABLED: "false"
+    volumes:
+      - /services/docker-compose/bind/data/etc:/etc/named
+      - /services/docker-compose/bind/data/var:/var/named
+    ports:
+      - "53:53/udp"
+      - "53:53/tcp"
+    deploy:
+      placement:
+        constraints: [node.role == manager]

```

Comment:
Does it need to be a manager node? Does UDP not pass through the load balancing?